packages/types/src/appearance.ts

```diff
@@ -33,6 +33,7 @@ type Shade =
   | '300'
   | '400'
   | '500'
+  | '525'
```

Comment:
Why did we introduced this particular shade? It's somewhat inconsistent with the rest.

Reply:
I'm going to remove this as I no longer need it.